projects/core/components/primitive-textfield/primitive-textfield.component.ts

```diff
@@ -77,7 +75,7 @@ export class TuiPrimitiveTextfieldComponent
 
     @Input()
     @tuiDefaultProp()
-    iconCleaner: TuiPrimitiveTextfieldOptions['iconCleaner'] = this.options.iconCleaner;
```

Comment:
I think we should probably deprecate it as input. Do you see a case where it would be dynamically changed?

Reply:
fixed